features/upgrade/README.md

```diff
@@ -1,178 +1,9 @@
-# Upgrading from rspec-rails-2.x to rspec-rails-3
+# Upgrading from rspec-rails 3.x to version 4
 
-For detailed information on the general RSpec 3.x upgrade process see the [RSpec
-Upgrade docs](https://relishapp.com/rspec/docs/upgrade).
+RSpec Rails 4 requires Ruby 2.2.
 
-There are several changes specific to `rspec-rails` to be aware of:
+Please also keep in mind that even though RSpec Rails 4 has soft support for Rails 4.2, it's recommended to use Rails 5+.
 
-- [Default helper files created in RSpec 3.x have changed](#default-helper-files)
+Other than that, there is nothing that needs to be done to upgrade from RSpec Rails 3 to version 4.0.
```

Comment:
I would rephrase this, I feel its important to emphasise the last part, and less so the first part.

```suggestion
RSpec Rails 4 is a major version under semantic versioning, it allowed us to change the supported Rails versions to 5 and 6 only. There are no changes required to upgrade to RSpec Rails 4 if you are using Rails 5 or 6.

If you are using Rails 4.2 you can use RSpec Rails 4, but note that support for it is not maintained, we consider this a breaking change hence the version change, and you must be on Ruby 2.2 as a minimum.
```

Reply:
Much better, thanks.